modules/nw-using-ingress-and-routes.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-using-ingress-and-routes_{context}"]
+= Using Ingress Controllers and routes
+
+The Ingress Operator manages Ingress Controllers and wildcard DNS.
+
+Using an Ingress Controller is the most common way to allow external access to
+an {product-title} cluster.
+
+An Ingress Controller is configured to accept external requests and proxy them
+based on the configured routes. This is limited to HTTP, HTTPS using SNI, and
+TLS using SNI, which is sufficient for web applications and services that work
+over TLS with SNI.
+
+Work with your administrator to configure an Ingress Controller
+to accept external requests and proxy them based on the
+configured routes.
+
+The administrator can create a wildcard DNS entry and then set up an ingress
+controller. Then, you can work with the edge Ingress Controller without
+having to contact the administrators.
+
+When a set of routes is created in various projects, the overall set of routes
+is available to the set of Ingress Controllers. Each Ingress Controller admits
+routes from the set of routes. By default, all Ingress Controllers
+admit all routes.
+
+Ingress Controllers that have permission to view all of the labels in all
+projects can select routes to admit based on the labels. This is called ingress
+controller sharding. This is useful when balancing incoming traffic load among a
+set of Ingress Controllers and when isolating traffic to a specific ingress
+controller. For example, company A goes to one Ingress Controller and company B
+to another.
+
+The Ingress Controller:
+
+* Has two replicas by default, which means it should be running on two nodes.
```

Comment:
Better to use `on two worker nodes` since all replicas are running on worker node by default.